futurex_openedx_extensions/dashboard/statistics/certificates.py

```diff
@@ -41,3 +42,61 @@ def get_certificates_count(
     )).values('course_org').annotate(certificates_count=Count('id')).values_list('course_org', 'certificates_count'))
 
     return dict(result)
+
+
+def get_learning_hours_count(
+    fx_permission_info: dict, visible_courses_filter: bool | None = True, active_courses_filter: bool | None = None
+) -> int:
+    """
+    Get the count of learning hours in the given tenants. The count is grouped by course_effort. Certificates
+    for admins, staff, and superusers are also included.
+
+    :param fx_permission_info: Dictionary containing permission information
+    :type fx_permission_info: dict
+    :param visible_courses_filter: Value to filter courses on catalog visibility. None means no filter.
+    :type visible_courses_filter: bool | None
+    :param active_courses_filter: Value to filter courses on active status. None means no filter.
+    :type active_courses_filter: bool | None
+    :return: Count of certificates per organization
+    :rtype: Dict[str, int]
+    """
+
+    def parse_course_effort(effort: str) -> float:
+        """Parses course effort in HH:MM format and returns total hours as a float."""
+        try:
+            parts = effort.split(':')
+            hours = int(parts[0])
+            minutes = int(parts[1]) if len(parts) > 1 else 0
+
+            if minutes >= 60:
+                raise ValueError('Minutes cannot be 60 or more.')
+
+            total_hours = hours + minutes / 60
+            return round(total_hours, 1)
+        except (ValueError, IndexError):
+            return settings.FX_DEFAULT_COURSE_EFFORT
+
+    result = list(
+        GeneratedCertificate.objects.filter(
+            status='downloadable',
+            course_id__in=get_base_queryset_courses(
+                fx_permission_info,
+                visible_filter=visible_courses_filter,
+                active_filter=active_courses_filter,
+            ),
+        ).annotate(
+            course_effort=Subquery(
+                CourseOverview.objects.filter(
+                    id=OuterRef('course_id')
+                ).values('effort')
+            )
+        ).annotate(
+            certificates_count=Count('id')
+        ).values('course_effort', 'certificates_count')
+    )
+
+    return sum(
```

Comment:
this is great, let's see the performance on production. We can cache it if it found it slowing down the dashboard